spec/components/find_interface/header/view_preview.rb

```diff
@@ -3,11 +3,11 @@
 module Find::Header
```

Comment:
hmmm this sticks out like a sore thumb :thinking:

Reply:
Yes, we need to rename the spec../find_interface.

Reply:
Done